analyses/survival-analysis/survival-analysis_HGG_DMG.Rmd

```diff
@@ -53,7 +53,9 @@ if (!dir.exists(plots_dir)) {
 }
 ```
 
-Declare input file paths and useful variables.
+Declare input file paths and useful variables
+
+Using v17 histologies file
```

Comment:
I'm a little bit worried about this phrasing, because the code itself is not tying it to a particular release. My inclination would be to put a note at the very top of the notebook that it has most recently been run with v17. However, I think it's very easy for this to get out of date (which is not ideal!). There are ways to use `params` here to tie this to v17 when run locally (or not in CI) but change the subdirectory to `testing` in CI, but I'm not sure how much we want to get into that here.